apps/sms/services/README.md

```diff
@@ -0,0 +1,91 @@
+# The layers in the Messages application
+
+## Terms explanation
+
+There are 3 layers whose interfaces managed by the Bridge (aka [threads.js](https://github.com/gaia-components/threads/)):
+
+* Services define high-level semantics to access the model. They are the only
```

Comment:
> be directly called by the views

Ok, I didn't want to mean that this was directly called as in a direct javascript call. Definitely this uses bridge.client.
I consider the bridge library as a technical help; so when I say "direct" I mean "without any functional intermediary object".

> So is Manager just a lib around some of the functionality, like DBManager, NotificationManager or more high level like ConversationManager that doesn't know anything aboout bridge and just helps with some stuff (e.g. wraps IDB work)?

Yeah, I'd rather do that unless we see it's a bad idea.

> And Wrapper is just a mix of Manager (wraps API work) and JService (uses bridge internally because it's hosted in iframe/window and can't be accessed directly from worker)?

Not sure I follow :) Wrapper is another type of bridge-managed service except it lives in a window. And it's really a "technical" artefact while managers are "functional" objects, that's why I give different names.